README.rst

```diff
@@ -62,12 +62,13 @@ a CLI prompt.
 (The different rho commands are covered more in the `Command Syntax & Usage`_
 section.)
 
-All the information that rho needs is stored in the data folder in
-the installed directory. All the auths are stored in the ``credentials``
-file. All the profiles are stored in the ``profiles`` file. The Ansible
-playbook is called ``rho_playbook.yml`` stored in the installed directory.
-The roles created during the scan are stored in the ``roles`` folder. These
-roles are used by ``rho_playbook.yml`` to perform the fact collection.
+All the information that rho needs is stored in the XDG_CONFIG_HOME/rho and
```

Comment:
I've seen these directories written as `$XDG_CONFIG_HOME/rho` (with the leading `$`) before. I think the version with the leading `$` might be more idiomatic, but have you seen it differently?

Reply:
updated with the leading `$`